man/systemd.unit.xml

```diff
@@ -232,6 +232,11 @@
     effective way to fully disable a unit, making it impossible to
     start it even manually.</para>
 
+    <para>Don't create unit files outside of the root partition and don't
+    create symlinks to outside of the root partition because systemd cannot
+    find and start them during early boot where mount units have not got
+    activated yet.</para>
```

Comment:
Indeed, this subject is not mentioned in the man page at all, afaics. But a few caveats:
- shouldn't this go in the section "Unit file load path" below?
- please make the language a bit more formal. We don't address the user as "you" in the surrounding text.
- it's not "root partition", because e.g. volatile systems will run with "/" on tmpfs, and all files in a (separate) "/usr" partition. So maybe "Unit files must be available when systemd starts, which means that they cannot be stored on file systems which are mounted later during boot. ...".

Reply:
Thanks for the comments. I found systemctl link is pointed to in "Unit file load path" section and I found the description I want is already present in the description of systemctl link. The description describes "when systemd starts" just as your suggestion. Thus, the current problem is that users doesn't reach the description. I update the pull request where I try to improve "Unit file load path" for users to reach systemctl link more easily.